api/services/ScheduledBuildHelper.js

```diff
@@ -52,7 +63,16 @@ const nightlyBuilds = () => Site.findAll({
       },
     ],
   },
-})
-  .then(sites => Promise.all(sites.map(site => buildSite(site))));
+};
+
+const nightlyBuilds = async () => {
+  const sites = await Site.findAll(query);
+
+  const builds = sites
+    .map(siteBranches)
+    .flat();
+
+  return Promise.allSettled(builds);
```

Comment:
use `allSettled` to ensure all of them complete in case one fails and report all of the information back.